src/controller/blob/controller.go

```diff
@@ -216,13 +217,26 @@ func (c *controller) Get(ctx context.Context, digest string, options ...Option)
 
 	opts := newOptions(options...)
 
-	params := blob.ListParams{
-		ArtifactDigest: opts.ArtifactDigest,
-		BlobDigests:    []string{digest},
-		ProjectID:      opts.ProjectID,
+	keywords := make(map[string]interface{})
+	if digest != "" {
+		ol := q.OrList{
+			Values: []interface{}{
+				digest,
```

Comment:
Seems there is only one value, use the exact match instead?

Reply:
both are ok, but to make logic coherence, I perfer to keep it.

Reply:
I prefer to change it to use exact match, the old method chose to use blob digests because there is no way to filter the blob by one digest.